core/src/main/java/org/apache/iceberg/schema/UnionByNameVisitor.java

```diff
@@ -57,6 +63,25 @@ public static void visit(UpdateSchema api, Schema existingSchema, Schema newSche
         new PartnerIdByNameAccessors(existingSchema));
   }
 
+  /**
+   * Adds changes needed to produce a union of two schemas to an {@link UpdateSchema} operation.
+   *
+   * <p>Changes are accumulated to evolve the existingSchema into a union with newSchema.
+   *
+   * @param api an UpdateSchema for adding changes
+   * @param existingSchema an existing schema
+   * @param caseSensitive when tfalse, the case of schema's fields are ignored
```

Comment:
Doc error and also I think we don't need to pass this through now that it is a field correct?

Reply:
the method is static and the field member is not accessible in the static context.